indexer/osm_editor.cpp

```diff
@@ -0,0 +1,109 @@
+#include "osm_editor.hpp"
+
+#include "indexer/feature_decl.hpp"
+#include "indexer/index.hpp"
+
+#include "std/map.hpp"
+#include "std/set.hpp"
+
+namespace osm
```

Comment:
And, if at some point in the future we will decide to move opening_hours from 3party, that "osm" library will be a good place for that.